src/transformState.js

```diff
@@ -5,7 +5,33 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key in actions) {
+    switch (actions[key].type) {
+
+      case 'addProperties':
+        Object.assign(state, actions[key].extraData);
+
+        break;
+
+      case 'removeProperties':
+        for (const property of actions[key].keysToRemove) {
+          delete state[property];
+        }
+
+        break;
+
+      case 'clear':
+        for (const property in state) {
+          delete state[property];
+        }
+
+        break;
+
+      default:
+
+        return state;
```

Comment:
Do we really need this blank line?
```suggestion
      default:
        return state;
```